### PR TITLE
Should be everything for recaptcha function

### PR DESCRIPTION
**Pull request by camidemery**

### CODE REVIEW

**Comment by leila-alderman** [comment]
Could you also update this PR with your PR that was just merged? Thanks!

**Comment by camidemery** [comment]
I mitigated Merge Conflicts. Is that what was needed to update the PR?

File: Gemfile
Comment (leila-alderman) [inline Gemfile:29]: Please specify versions for these gems.

File: config/initializers/recaptcha.rb
Comment (leila-alderman) [inline config/initializers/recaptcha.rb:6]: It can be quite dangerous to post secret keys onto GitHub, so I can't merge this PR as-is. 

Adding these as environment variables is probably the most correct solution, but we can do this another way that should be easier. 

1. Add this file to your `.gitignore` file. 
2. Create a new file named `config/initializers/recaptcha_example.rb`.
3. In this new file, copy and paste the content from this file, but replace both keys with something like `"example"`. 
4. You will have to actually send your teammates the ignored file in order for the code to work for them.

**Review by leila-alderman — CHANGES_REQUESTED** [review]
Thanks for updating this PR with the already merged changes!

A few changes are needed here before this PR can be merged. Please let me know if you have any questions!

**Pull request closed without merging**
